### ISSUE TITLE
Rake task erd does not show up in rake -T

### ISSUE THREAD

**Issue by sebroeder** [voormedia/rails-erd#235]
I have a fresh Rails 5.0.1 (Ruby 2.3.3) project and put `rails-erd` in the `development` group of my Gemfile:

```ruby
group :development do
  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  gem 'listen', '~> 3.0.5'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
  # Entity-Relationship Diagrams for Ruby on Rails
  gem 'rails-erd'
end
```

I run `bundle` to install the gem.

I want to see the `erd` task in the list provided by `bundle exec rake -T`, but it does not show up. I tried all the alternative invocations `bundle exec rails -T`, `bin/rails -T` and `bin/rake -T` with the same result.

I can actually run the task with `bundle exec rake erd` without a problem, but it does not show up in the task list.

I checked for issues related to Rake tasks on the Ruby on Rails GH issues and could not find anything related.

When I moved the gem out of the `development` group in the Gemfile and into the "global" namespace, it **did** show up in the task list:

```ruby
group :development do
  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  gem 'listen', '~> 3.0.5'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end

# Entity-Relationship Diagrams for Ruby on Rails
gem 'rails-erd'
```

I have no clue why this is happening. Any ideas?

**Comment by kerrizor** [comment]
IIRC a task won't show up in the output of `rake -T` unless the `desc` is set. Should be a straight-forward fix (or at least an interesting bit of research for someone!)

**Comment by sebroeder** [comment]
I am pretty sure this is not the issue here, because there is a description for the default test at https://github.com/voormedia/rails-erd/blob/master/lib/rails_erd/tasks.rake#L61 and this would not explain why it works from outside the `development` group of the Gemfile.

https://github.com/rails/rails/pull/29029 reminded me of our problem here, though. So maybe it was an issue in Rails and is now fixed in the latest versions. I‘ll report back as soon as I find the time to try this out.

Thank you @kerrizor for your efforts to work through the issues. 🌻

**Comment by sebroeder** [comment]
@kerrizor I can confirm that this was a Rails bug and got fixed in rails/rails#29029 for Rails >= 5.0.3 and Rails >= 5.1.1 🎉

**Comment by kerrizor** [comment]
Thanks for following up!
